src/adaptation/stepsize.jl

```diff
@@ -1,10 +1,22 @@
 ### Mutable states
+"""
+    Dual Averaging state
```

Comment:
```suggestion
    DAState{T<:AbstractScalarOrVec{<:AbstractFloat}}
```
Thanks for spotting that this and a few other types needed docstrings. I know that the description "Dual Averaging state" is more informative than the name "DAState", but the Julia docstring [docs](https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation) do request that the first line would always be the actual signature of the function/struct.

Reply:
Agreed, there are still tons of docstrings that need such formation, but let's focus on the printing stuff here and leave these docstrings for another thorough update.